content/docs/concepts/architecture/_index.md

```diff
@@ -56,19 +80,32 @@ Governance operations are used to modify protocol parameterisation set in the ge
 
 For all parameter definitions and the subset of modifiable parameters see the [Protocol Parameter](/reference/protocol/) reference.
 
-### Staking
+#### State finalisation
+The Autonity Protocol Contract manages state finalisation, maintaining [system state](/glossary/#system-state). Contract logic triggers:
+
+- block finalisation
+- at epoch end:
+  - [selection of a new consensus committee](/concepts/architecture/#committee-selection) for the following epoch
```

Comment:
The order of operation is different now https://github.com/autonity/autonity/blob/8183c96276bfc0764ad59b31cde423d6245329c0/autonity/solidity/contracts/Autonity.sol#L587

Reply:
Yes. I updated the `finalize()` function but not here.  Corrected in commit 09a45fd2396e397f487814709f6a541ddb4e4c03